doc/source/user_guide/numpy_images.rst

```diff
@@ -229,27 +228,27 @@ in a different order, even if the number of operations is the same:
     Speedup: 8.6x
 
 
-When the last/rightmost dimension becomes even larger, the
-speedup is even more dramatic. It is worth thinking about
-*data locality* when writing algorithms. In particular, know that
-scikit-image uses C-contiguous arrays unless otherwise specified.
+When the last/rightmost dimension becomes even larger the speedup is
+even more dramatic. It is worth thinking about *data locality* when
+you are developing algorithms. In particular, you should know that
```

Comment:
~~you are~~

Reply:
~~you should~~. Keep it impersonal

Reply:
(Clarification: I feel like "you should" sounds kinda bossy, as opposed to a gentle recommendation. But to be clear, "know that..." by itself also sounded a bit jarring here.

Reply:
Maybe just, "in particular, scikit-image uses C-contiguous arrays by default"